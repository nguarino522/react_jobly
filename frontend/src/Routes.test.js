import React from "react";
import { render } from "@testing-library/react";
import Routing from "./Routes";
import { MemoryRouter } from "react-router";
import { UserProvider } from "./testUtils";

it("renders without crashing", function () {
  render(
      <MemoryRouter>
        <UserProvider>
          <Routing />
        </UserProvider>
      </MemoryRouter>,
  );
});

it("matches snapshot", function () {
  const { asFragment } = render(
      <MemoryRouter>
        <UserProvider>
          <Routing />
        </UserProvider>
      </MemoryRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
