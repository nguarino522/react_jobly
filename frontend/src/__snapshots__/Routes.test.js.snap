// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="Homepage"
    >
      <div
        class="container text-center"
      >
        <h1
          class="mb-4 font-weight-bold"
        >
          Jobly
        </h1>
        <p
          class="lead"
        >
          All the jobs in one, convenient place.
        </p>
        <h2>
          Welcome Back, testuser!
        </h2>
        <p>
          <a
            class="btn btn-primary font-weight-bold m-2"
            href="/companies"
          >
            Companies
          </a>
          <a
            class="btn btn-primary font-weight-bold m-2"
            href="/jobs"
          >
            Jobs
          </a>
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
